src/main/java/org/jabref/gui/push/AbstractPushToApplication.java

```diff
@@ -181,4 +182,40 @@ public Optional<KeyBinding> getKeyBinding() {
             return Optional.of(KeyBinding.PUSH_TO_APPLICATION);
         }
     }
+
+    /**
+     * This function is to jump to a specific line due to different editor.
+     *
+     */
+    protected void jumpToLine(Path fileName, int line, int column) {
+        commandPath = preferences.getPushToApplicationPreferences().getCommandPaths().get(this.getDisplayName());
+
+        // Check if a path to the command has been specified
+        if (StringUtil.isNullOrEmpty(commandPath)) {
+            notDefined = true;
+            return;
+        }
+
+        // Execute the command
+        String[] command = jumpString(fileName, line, column);
+
+        try {
+            ProcessBuilder processBuilder = new ProcessBuilder(command);
+            processBuilder.start();
+        } catch (IOException e) {
+            // Use robust logging instead of printStackTrace()
+            LOGGER.error("Could not open TeXstudio at the specified location.", e);
+
+            // Show an error dialog to the user
+            dialogService.showErrorDialogAndWait("Error", "Could not open TeXstudio at the specified location.");
```

Comment:
Use localization. Paramterize the tool using `%0`.  See https://devdocs.jabref.org/code-howtos/localization.html for details.